README.md

```diff
@@ -41,3 +41,20 @@ n = 10**5
 sleep_durations = [random.uniform(0, 5.0) for _ in range(n)]
 asyncio.run(run_multiple_sleeps(sleep_durations=sleep_durations))
 ```
+
+## Demo
+A complete demo of `tqdm_publisher` can be found in the `demo` directory, which shows how to forward progress updates from the same `TQDMPublisher` instance to multiple clients.
+
+To run the demo, first install the dependencies:
+```bash
+pip install tqdm_publisher[demo]
+```
+
+Then, run the base CLI command to start the demo server and client:
+```bash
+tqdm_publisher
```

Comment:
```suggestion
tqdm_publisher_demo
```

Thanks, this is looking a lot better now!

I'd still recommend attaching "demo" to the CLI verbage to make it clear this is not in and of itself a CLI package (like, running `tqdm_publisher` on terminal won't magically make a users existing tqdm calls route through our new class)

Reply:
Oops, you beat me to it lol

Reply:
Knew it was coming ;)